srcpkgs/gpgme/template

```diff
@@ -81,3 +84,24 @@ gpgmeqt-devel_package() {
 		vmove usr/lib/cmake/QGpgme
 	}
 }
+
+gpgme-python3_package() {
+	short_desc+=" - Python binding"
+	depends="gpgme"
+	pkg_install() {
+		vmove "usr/lib/python*"
+		_sitepath="${PKGDESTDIR}/${py3_sitelib}"
+		_dotegg=$(echo "${_sitepath}"/gpg-"${version}"-*.egg)
+		_dotegginfo="$_dotegg"-info
+		mkdir -p $_dotegginfo
+		mv "${_dotegg}"/gpg "${_sitepath}"/
+		mv "${_dotegg}"/EGG-INFO/* "${_dotegginfo}"
+		rm -r ${_dotegg}
```

Comment:
This is wrong; not only is it more complicated than it needs to be, but it uses the wrong egg-info name. You should also declare the variables you create local. At a minimum, you can reduce this dance to somthing like:

```suggestion
		local _sitelib="${PKGDESTDIR}/${py3_sitelib}"
		vmove "${py3_sitelib}"
		mv "${_sitelib}"/gpg-*.egg/gpg "${_sitelib}"
		mv "${_sitelib}"/gpg-*.egg/EGG-INFO \
			"${_sitelib}/gpg-${version}-py${py3_ver}.egg-info"
```

However, it would be better still to figure out how to install the Python package the right way.

Reply:
I think it is better to build python package with option `--wheel`